pyiron_base/database/sqlcolumnlength.py

```diff
@@ -0,0 +1,11 @@
+DEFAULT_STR_LENGTH = 255
+PROJECT_PATH_STR_LENGTH = DEFAULT_STR_LENGTH
+PROJECT_STR_LENGTH = DEFAULT_STR_LENGTH
+JOB_STR_LENGTH = DEFAULT_STR_LENGTH
+SUBJOB_STR_LENGTH = DEFAULT_STR_LENGTH
+CHEMICALFORMULA_STR_LENGTH = 50
+STATUS_STR_LENGTH = 20
+HAMILTON_STR_LENGTH = 20
+HAMVERSION_STR_LENGTH = 50
+USERNAME_STR_LENGTH = 20
+COMPUTER_STR_LENGTH = 100
```

Comment:
I moved the variable definition to a separate file to prevent requiring `sqlalchemy` on `pyiron_base` installations without central database.